### DISCUSSION TITLE
2022 年度总结: 我的多重宇宙

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#392] [Announcements]
# 2022 年度总结: 我的多重宇宙

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2022/02-2022-review/

**Comment by gaocegege** [comment]
写的真好！

**Comment by Rouzip** [comment]
这样的生活太棒了吧！希望有一天能过上这样的生活！（被虚假开源骗去搬砖的我掩面哭泣orz

**Comment by zbtzbtzbt** [comment]
一年改了30w行代码，太猛了，我的超人！xuanwogiegie！

**Comment by songpengwei** [comment]
好奇一下博客文章统计用的是什么平台？看起来不像谷歌分析？

**Reply by Xuanwo** [reply]
开源的 https://plausible.io/ ～

**Comment by justinyanme** [comment]
送出一个小火箭~🚀楼主真棒

**Comment by lmmsoft** [comment]
泡澡那段爱了，看了就感觉好放松啊~

**Comment by J1a-wei** [comment]
赞，宝藏博客！

**Comment by stan2133** [comment]
该写去CMU讲课的事儿了
